src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -159,6 +208,36 @@ export const ArtistInfoFragmentContainer = createFragmentContainer(
       exhibition_highlights(size: 3) {
         ...SelectedExhibitions_exhibitions
       }
+      collections
```

Comment:
It might be worth a comment here explaining that all this stuff has been added to the fragment for the exist-y check for the button. This winds up being consolidated and deduped with the data requirement from the fragment here:  https://github.com/artsy/reaction/blob/6e3144e252f646cf1d45ea2b492b87752948c0b8/src/Components/Artist/MarketInsights/MarketInsights.tsx#L219

If one of the fragments is then subsequently changed, it could lead to overfetching so it might be worth a comment!